### ISSUE TITLE
Suspicious link text

### ISSUE THREAD

**Issue by r-leyshon** [banksad/returns_to_education#8]
**<h3>WCAG2.1 accessibility issues</h2>**

**Type**: Alert

**Source**: localhost
**Short desc**:  Don't use 'here' for link text.

**Long desc**: 

What It Means
Link text contains extraneous text or may not make sense out of context.
Why It Matters
Links, which are often read out of context, should clearly describe the destination or function of the link. Ambiguous text, text that does not make sense out of context, and extraneous text (such as "click here") can cause confusion and should be avoided.
How to Fix It
Where appropriate, reword the link text so that it is more descriptive of its destination when read out of context. Remove any extraneous text (such as "click here").
The Algorithm... in English
A link (including alt text of linked images) contains the phrase "click here" or "click", or the link text is "click here", "here", "more", "more...", "details", "more details", "link", "this page", "continue", "continue reading", "read more", or "button".

**WCAG ref**:
* [2.4.4 Link Purpose (In Context) (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.4)

**Code ref**:

```
<a href="https://github.com/banksad/returns_to_education" target="_blank" rel="noopener noreferrer">
here
</a>
```
